lib/llvmopencl/ParallelRegion.cc

```diff
@@ -498,9 +499,9 @@ ParallelRegion::AddParallelLoopMetadata(
         continue;
       }
 
-#ifdef LLVM_OLDER_THAN_13_0
+#if LLVM_VERSION_MAJOR < 13 && !(LLVM_VERSION_MAJOR == 12 && LLVM_VERSION_MINOR >= 0 && LLVM_VERSION_PATCH >= 1)
```

Comment:
Please mind the row length.

Reply:
Good point. Used provided formatter to format this line.